src/main/java/tr/com/srdc/cda2fhir/transform/ValueSetsTransformerImpl.java

```diff
@@ -690,29 +690,19 @@ public Coding tRoleCode2PatientContactRelationshipCode(String cdaRoleCode) {
 			return null;
 		
 		Coding fhirPatientContactRelationshipCode = new Coding();
-		fhirPatientContactRelationshipCode.setSystem("http://hl7.org/fhir/patient-contact-relationship");
+		fhirPatientContactRelationshipCode.setSystem("http://hl7.org/fhir/v2/0131");
 		String code = null;
 		String display = null;
 		
 		switch(cdaRoleCode.toLowerCase()) {
 			case "econ": // emergency contact
-				code="emergency"; display = "Emergency"; break;
 			case "ext": // extended family member
-			case "fammemb": // family member
-				code = "family"; display = "Family"; break;
```

Comment:
Can we keep descriptions "// family member" in the code? They are useful.

Reply:
I guess I considered Next of kin to only be brother/sister? That's exactly why I made this PR. Should all the family related codes be next of kin? What about father in law/mother in law? Does next of kin specifically refer to the closest relative, or are all immediate family members considered "next of kin"? 

I think this is a pretty good example of why we need to get jolt up and running, along with whatever we'll be using to turn it into a flat file so we can get feedback from UPMC.